modules/test/playwright/tests/frontend-data-set-web/clayTableWidth.spec.ts

```diff
@@ -0,0 +1,98 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {applicationsMenuPageTest} from '../../fixtures/applicationsMenuPageTest';
+import {commercePagesTest} from '../../fixtures/commercePagesTest';
+import {dataApiHelpersTest} from '../../fixtures/dataApiHelpersTest';
+import {loginTest} from '../../fixtures/loginTest';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	applicationsMenuPageTest,
+	commercePagesTest,
+	dataApiHelpersTest,
+	loginTest()
+);
+
+test('LPD-31378 Clay Table Width', async ({
+	apiHelpers,
+	applicationsMenuPage,
+	page,
+}) => {
+	await test.step('Create commerce site', async () => {
+		const site = await apiHelpers.headlessSite.createSite({
+			name: 'Minium',
+			templateKey: 'minium-initializer',
+			templateType: 'site-initializer',
+		});
+
+		apiHelpers.data.push({id: site.id, type: 'site'});
+
+		await applicationsMenuPage.goToSite('Minium');
+	});
+
+	await test.step('Create account', async () => {
+
+		const accountButton = page.getByRole('button', {
+			name: 'Select Account & Order',
+		});
+
+		await accountButton.waitFor({state: 'visible'});
+		await accountButton.click();
+
+		const newAccountButton = page.getByRole('button', {
+			name: 'Create New Account',
+		});
+
+		await newAccountButton.waitFor({state: 'visible'});
+		await newAccountButton.click();
+
+		const accountName = page.locator('input[name="accountName"]');
+
+		await accountName.waitFor({state: 'visible'});
+		await accountName.fill('test');
+
+		await page.getByRole('button', {name: 'Create', exact: true}).click();
+	});
+
+	await test.step('Add transmission to shopping cart', async () => {
+
+		const transmissionButton = page
+			.locator('#wwxc_column_2d_2_1_add_to_cart')
+			.getByRole('button', {name: 'Add to Cart'});
+
+		await transmissionButton.waitFor({state: 'visible'});
+		await transmissionButton.click();
+
+		const cart = page.getByRole('button', {name: '1'});
+
+		await cart.waitFor({state: 'visible'});
+		await cart.click();
+
+	});
+
+	await test.step('Check SKU width length', async () => {
+
+		const viewDetails = page.getByRole('button', {name: 'View Details'});
+
+		await viewDetails.waitFor({state: 'visible'});
+		await viewDetails.click();
+
+		const SKU = page
+			.getByTestId('visualization-mode-table')
+			.getByText('SKU');
+
+		await SKU.waitFor({state: 'visible'});
+
+		const SKUCell = await SKU.boundingBox();
+
+		const SKUWidth = SKUCell.width;
+
+		expect(SKUWidth).toBeGreaterThan(70);
```

Comment:
`SKU`, `cart`, `viewDetails`, `accountName` etc, these are not good names for variables. A variable name needs to reflect what is value _is_, what it represents. If it's clear from the context, it can be generic, like `input`, `width`, `site`, `button`. If you want it more specific, then you append a prefix that makes it more specific, like `skuInput`, `commerceSite`, `saveButton` etc.

Reply:
I'll be more careful with this, thank you!